internal/clients/image_builder/client.gen.go

```diff
@@ -15,7 +15,6 @@ import (
 	"strings"
 
 	"github.com/deepmap/oapi-codegen/pkg/runtime"
-	"github.com/go-chi/chi/v5"
 )
```

Comment:
Also the generated code by @adiabramovitch was generated with "server" code. We don't actually need that.